modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -1,61 +1,51 @@
-= Client ID enforcement
+= Client ID Enforcement Policy
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 [width="100%", cols="5,15"]
 |===
->s| Policy Name | Client ID enforcement
+>s| Policy Name | Client ID Enforcement 
 >s|Summary      | Allows access only to authorized client applications
 >s|Category | Security
->s| Since Mule Version | 3.8.0
+>s| Available from Mule | v3.8.0
 .4+>.^s| Returned Status Codes
 |401 - Unauthorized or invalid client application credentials
 |500 - Bad response from authorization server, or WSDL SOAP Fault error
 |===
 
-The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request.
+The Client ID Enforcement policy restricts access to a protected resource by allowing requests only from registered client applications. The policy ensures that the client credentials sent on each request have been approved to consume the API.
 
-The policy validates the Client ID and, optionally, the Client Secret, of a client application created within an Anypoint Platform Organization.
+When a client application is registered in Anypoint Platform, a pair of credentials consisting of a client ID and client secret is generated. When the client application requests access to an API, a contract is created between the application and that API. An API that is protected with a Client ID Enforcement policy is accessible only to applications that have an approved contract.
 
-IMPORTANT: This policy will not validate successfully any client application credentials generated outside of the Anypoint Platform context.
+When you apply a Client ID Enforcement policy, access to your API is tracked by reporting the client ID along with the analytics events. You can configure Mule runtime engine (Mule) to encrypt all sensitive information that pertains to policies,contracts, and initialization data.
 
-NOTE: The set of Client ID and Client Secret are also known as credentials.
-
-= Analytics and  access tracking
-If you need to track client application access to your API, apply a policy that directly or indirectly makes sure that client ID is provided in the HTTP request (ie Client ID enforcement type policy).
-
-= Indirect client ID enforcement
-There are other provided policies that enforce client application credentials. For example:
+Default policies that internally enforce client application credentials include:
 
 * xref:rate-limiting-and-throttling-sla-based-policies.adoc[Rate Limiting - SLA-Based Policies]
```

Comment:
Beware that this may change after the unification you will do later on.

Reply:
Noted! Thanks!